app/controllers/orders_controller.rb

```diff
@@ -22,14 +25,29 @@ def create
 
     # Send to the confirmation page aka orders#show
     check_inventory(@order)
+
     if @order.save
-      reduce_inventory(@order)
-      redirect_to complete_order_path
+      # get quotes
+      address = { country: @order.country, state:  @order.state, city:  @order.city, zip:  @order.zip }
+      num_items = @order.order_items.count
+      @shipping_response = ShipItWrapper.get_quotes(address, num_items)
+      @rates =  JSON.parse(@shipping_response.body)
+      @status = @shipping_response.code
+      @order.update(rates: @rates)
+      render :new
     else
       render :new
     end
```

Comment:
since you render new in either case, you can pull that pull outside of the conditional